### ISSUE TITLE
Name Anchor links work in Editor, but do not on actual page

### ISSUE THREAD

**Issue by uxinkc** [Requarks/wiki#1513]
**Describe the bug**
When I create a named anchor to scroll to a specific section on the page, the named anchor works in the Wiki Editor preview panel, but when saved and viewing the actual page, the named anchor does not work.

**To Reproduce**
1. Navigate to page
2. Click on Edit Page button in lower right corner
3. Add a named anchor `<a name="myAnchor">My Anchor</a>` 
4. Further down the page add `<a href="#myAnchor">Got to Anchor</a>`

**Screen shots**
![Screen Shot 2020-02-21 at 8 39 18 AM](https://user-images.githubusercontent.com/53310223/75043455-bc043f80-5485-11ea-8610-711b0a3b384d.png)

![Screen Shot 2020-02-21 at 8 40 29 AM](https://user-images.githubusercontent.com/53310223/75043523-d63e1d80-5485-11ea-8573-d3cfa1792563.png)

![Screen Shot 2020-02-21 at 8 41 47 AM](https://user-images.githubusercontent.com/53310223/75043651-0ab1d980-5486-11ea-90f6-553f79b9bd44.png)

![Screen Shot 2020-02-21 at 8 42 33 AM](https://user-images.githubusercontent.com/53310223/75043727-20270380-5486-11ea-9cec-865ba8e53f25.png)

**Expected behavior**
Scrolling up to the named anchor.

**Additional context**
I have tried using both HTML and Markdown syntax, but neither work in the actual page, but BOTH work in the Editor View.

**Comment by Smankusors** [comment]
how about using `id` property instead of `name` property?

**Comment by uxinkc** [comment]
Yeah, it seem I have to go that route, but I prefer to use the name attribute as intended, so that the ID attribute can always be freed up for other purposes. It still is a "bug", since name works in the Editor Preview, but not the actual page.  A User wouldn't assume something only works in the Editor portion of the application and NOT in the actual page.

**Comment by TogarUshindi** [comment]
Name Anchors would be appreciated since we're batch converting several web 1.0 documents using them.

**Comment by TimoKruth** [comment]
Verified against current v2 `main` (`6f042e97`, v2.5.314): I can no longer reproduce this bug in the current rendering pipeline.

I rendered the exact pattern from the report. The saved page HTML preserves both `<a name="myAnchor">` and `<a href="#myAnchor">`; the HTML security sanitizer also preserves the `name` attribute, and HTML Core explicitly leaves hash-only links unchanged.

Classification: appears fixed on current v2. (`id` anchors are still the more modern HTML form, but `name` works here.)
